packages/quantic/force-app/main/default/lwc/searchBoxStyle/searchBoxStyle.css

```diff
@@ -31,18 +31,21 @@
   height: 48px;
 }
 
-.searchbox__clear-button {
-  height: 48px;
-}
-
 .searchbox__submit-button:focus {
   border: none;
   border-top-right-radius: 0.25rem;
   border-bottom-right-radius: 0.25rem;
 }
 
+.searchbox__clear-button-container {
```

Comment:
This fix seems to apply only in the expandable search box of the Quantic Search Box component.
It's not applied to the expandable search box of the Quantic Standalone Search Box component cause this last the component this new CSS class is not being used in the HTML thus I guess the fix was not applied there.

I know you included a screenshot of the Standalone search box in the description of the PR but was it an expandable one?

Reply:
you are right, forgot to push the changes to the standalone html....

here : 

<img width="507" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/c45ccf61-8ae8-40de-a32a-6a7cad8cbb42">